variants/frontend-base/sentry/template.rb

```diff
@@ -49,4 +49,4 @@
 
 gsub_file "config/initializers/content_security_policy.rb",
           /# policy.report_uri ".+"/,
-          'policy.report_uri ENV["SENTRY_CSP_HEADER_REPORT_ENDPOINT"]'
+          'policy.report_uri(ENV["SENTRY_CSP_HEADER_REPORT_ENDPOINT"]) if ENV["SENTRY_CSP_HEADER_REPORT_ENDPOINT"]'
```

Comment:
I've hit this occasionally and it's quite annoying to **force** this ENV var to exist before the app will boot.